streaming/java/BUILD.bazel

```diff
@@ -253,7 +228,11 @@ genrule(
     cmd = """
         WORK_DIR="$$(pwd)"
         # Copy native dependencies.
-        NATIVE_DEPS_DIR="$$WORK_DIR/streaming/java/streaming-runtime/native_dependencies/"
+        OS_NAME="linux"
+        if [[ "$$OSTYPE" =~ darwin* ]]; then
```

Comment:
Please handle Windows everywhere across your PR (even if it it's just to produce an error in that case). Prefer to use `case` over `if [[ ]]`:

```
case "$${OSTYPE}" in
  linux*) ...;;
  darwin*) ...;;
  msys) ...;;
esac
```